kedro-telemetry/README.md

```diff
@@ -1,4 +1,4 @@
-# Kedro-Telemetry
+# Kedro-Telemetry!
```

Comment:
Are we trying to make kedro-telemetry look more exciting? 🤔

Reply:
hahah 😂  no I just pushed some random changes to kick-off the builds to check they're running. Sadly, `kedro-airflow` seems broken.. I'll revert these later.

Reply:
Ahh ok 😀  I thought it was unlikely to be a rebranding exercise...